client/packages/coldchain/src/Sensor/Components/NonEditableSensorTab.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { FC } from 'react';
+import { SensorFragment } from '../api';
+import { Grid } from '@mui/material';
+import {
+  TextWithLabelRow,
+  useTranslation,
+} from '@openmsupply-client/common/src';
+
+interface NonEditableSensorTabProps {
+  draft: SensorFragment;
+}
+
+export const NonEditableSensorTab: FC<NonEditableSensorTabProps> = ({
+  draft,
+}) => {
+  const t = useTranslation('coldchain');
+
+  return (
+    <Grid container display="flex" flex={1} flexDirection="column" gap={1}>
+      <TextWithLabelRow
+        label={t('label.cce')}
+        labelWidth="140px"
+        text={''}
+        textProps={{
+          textAlign: 'start',
+          paddingLeft: '50',
+          paddingTop: '4',
+          paddingBottom: '4',
+        }}
+        labelProps={{ sx: { padding: '4', textAlign: 'end' } }}
+        sensorStyle={true}
```

Comment:
This feels like a clumsy way to pass through styling specific to this component.

I wondered if a different TextWithLabelRow component would be better with different styling defaults?

I noticed we have two InputWithLabelRow components - perhaps could do something similar here?